src/panels/lovelace/mixins/element-click-mixin.js

```diff
@@ -12,8 +13,72 @@ import computeStateName from "../../../common/entity/compute_state_name";
 export default dedupingMixin(
   (superClass) =>
     class extends NavigateMixin(EventsMixin(superClass)) {
-      handleClick(hass, config) {
-        const tapAction = config.tap_action || "more-info";
+      registerMouse(config) {
+        let ripple = null;
+
+        const startAnimation = async () => {
+          if (ripple) return;
+          ripple = document.createElement("mwc-ripple");
+          this.shadowRoot.querySelector("div").appendChild(ripple);
```

Comment:
One final thought. Could we wrap the ripple in a div?

Just really trying to have to avoid an extra div having to be added to each element